### ISSUE TITLE
Inform the user about the current in-flight payments

### ISSUE THREAD

**Issue by hackaugusto** [raiden-network/raiden#2412]
## Problem Definition

if a payment takes too long a user may try a second time, unaware the previous has not completed yet, which may result in the same payment being made twice. To avoid this the UI should:

- [ ] Inform the user about the current payments which are in-flight
- [ ] Ask for a confirmation if the user tries a second payment for the same target and value is tried.

**Comment by kelsos** [comment]
@hackaugusto  I did some investigation and it seems that

1. The angular http client retries timeouts by default.
2. The timeout is currently at 2 minutes.

Based on the discussion we had about this we will start with the following actions:

1. Remove automatic retries for the requests
2. Increase the timeout for sending payments (and how much) 

Is that right?

**Comment by hackaugusto** [comment]
Yes, if we can avoid retries it would be best. How long is a hard question, IMO as large as it can be without making for a bad user experience

**Comment by LefterisJP** [comment]
Related: https://github.com/raiden-network/raiden/issues/2436
